input_cloud: "/map_nodelet/pointcloud"
step_maybe: "/staircase_detection/steps_centroid"
output_steps: "/staircase_detection/pointcloud/steps"
step_bounding_box: "/staircase_detection/pointcloud/bounding_box"
extract_bool: true
verbose: false
delta_angle: 0.087
z_passthrough: 0.002
distance_threshold: 0.12
cluster_tolerance: 0.1

